### PR TITLE
Added Codiad to the list

### PR DESCRIPTION
**Pull request by pilotpirxie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
...
 - [Browxy](https://www.beta.browxy.com) - Run Java, C, C++, C#, Python, or PHP with some syntax highlighting, input arguments, and publish to a public URL.
 - [Superblocks Lab](https://superblocks.com/lab) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/SuperblocksHQ/studio) - Run Solidity with a built-in browser Ethereum blockchain VM, Metamask integration (deployments to Testnet/Mainnet), transaction logger and live code your WebApp, powered by Monaco Editor.
+- [Codiad](https://github.com/codiad/codiad) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/codiad/codiad) - Codiad is a PHP web-based IDE framework with a small footprint and minimal requirements.
```
Comment (styfle) [inline README.md]: Change the main url to http://demo.codiad.com so that user's can try it out.

Comment (styfle) [reply README.md]: What languages does this IDE support?

The description here does not comply with the [contributing guidelines](https://github.com/styfle/awesome-online-ide/blob/master/contributing.md)

**Comment by styfle** [comment]
Hi @pilotpirxie thanks for the PR!

I left a couple comments--I'm not sure this meets the requirements of IDE as defined in the readme.

> An (IDE) is a software application that provides comprehensive facilities to computer programmers for software development. This typically includes a text editor, syntax highlighting, file explorer, debugger, version control, and build/run/deploy options. Another defining feature of an IDE is some form of intelligent code completion, sometimes called intellisense.

I see the first 3 things: text editor, syntax highlighting, file explorer.
But the other featrues such as debugger, version control and build/run/deploy are missing.

**Comment by pilotpirxie** [comment]
> Hi @pilotpirxie thanks for the PR!
> 
> I left a couple comments--I'm not sure this meets the requirements of IDE as defined in the readme.
> 
> > An (IDE) is a software application that provides comprehensive facilities to computer programmers for software development. This typically includes a text editor, syntax highlighting, file explorer, debugger, version control, and build/run/deploy options. Another defining feature of an IDE is some form of intelligent code completion, sometimes called intellisense.
> 
> I see the first 3 things: text editor, syntax highlighting, file explorer.
> But the other featrues such as debugger, version control and build/run/deploy are missing.

Hey styfle, 😃 
Codiad supports git for versioning control and can emulate and execute shell commands. By default those features are unavailable. They can be installed as free extensions from the plugin market. All that because most of the users have disabled either shell_exec() or exec() functions by default.

**Comment by styfle** [comment]
So executing arbitrary shell commands means this could potentially run any language?

If that's the case, the description should start with "Run any language"

Also the icons need to be flipped so that the github logo is on the left and the docker logo is on the right.

Thanks!

File: README.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
...
 - [Browxy](https://www.beta.browxy.com) - Run Java, C, C++, C#, Python, or PHP with some syntax highlighting, input arguments, and publish to a public URL.
 - [Superblocks Lab](https://superblocks.com/lab) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/SuperblocksHQ/studio) - Run Solidity with a built-in browser Ethereum blockchain VM, Metamask integration (deployments to Testnet/Mainnet), transaction logger and live code your WebApp, powered by Monaco Editor.
+- [Codiad](http://demo.codiad.com) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/codiad/codiad) <img title="Self Hosted" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/docker.svg" /> - Run most of languages with shell execution after installing terminal extension. Codiad is a PHP web-based IDE framework with a small footprint and minimal requirements.
```
Comment (styfle) [inline README.md]: The description can be more consise based on [contributing guidelines](https://github.com/styfle/awesome-online-ide/blob/master/contributing.md)

Let's use the following:

> Run most of languages on a self-hosted, resource-constrained server with plugins and shell execution.

**Pull request merged**